pkg/utils/utils.go

```diff
@@ -29,3 +43,69 @@ func FormatSize(size int64) string {
 
 	return fmt.Sprintf("%.2f %s", formattedSize, suffixes[count])
 }
+
+// CreatedTime is used to show the time from creation to now.
+func CreatedTime(created string) (s string) {
```

Comment:
What if I input an invalid string as a parameter in this func?

if `asdfghjkl`, what will happen? I think we need to take some exception cases into consideration to improve rubustness.

Reply:
Maybe we should check the validity of input.
But if we use `strconv.FormatInt(time.Now().Unix(), 10)` as input(which must be),the validity will  always be ok.  WDYT? @allencloud

Reply:
We do need the check. We need to handle exceptions.

Reply:
Ok,i will refactor the function and fix the bug.